tests/page-objects/users/add-user-modal.po.js

```diff
@@ -38,26 +42,84 @@ const getCancelButton = () => {
   return element(by.className('btn cancel'));
 };
 
+const waitForTranslation = (timeout = 50000) => {
+  const description=element.all(by.css('.help-block.ng-scope')).first();
```

Comment:
Can this be made a bit more generic to work for any translation checking? 

`const usernameHelp = 'This is what you will use to log in to the app.';`
` if(text === 'user.username.help'){`